protocol/alerthash/alerthash.go

```diff
@@ -29,6 +29,7 @@ type BotInfo struct {
 
 // ForBlockAlert calculates the hash for the block alert.
 func ForBlockAlert(inputs *Inputs) string {
+	sort.Strings(inputs.Finding.Addresses)
```

Comment:
any reason for doing this?

Reply:
this should have been like this from the start - can cause non-deterministic hashes if the array isn't sorted